src/org/zaproxy/zap/extension/exportreport/FileChooser/Utils.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+package org.zaproxy.zap.extension.exportreport.FileChooser;
+
+import java.io.File;
+import java.util.Locale;
+import javax.swing.ImageIcon;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.Constant;
+import org.zaproxy.zap.extension.exportreport.Export.ExportReport;
+
+/*
+ * AUTHOR		:	GORAN SARENKAPA - JordanGS
+ * SPONSOR		:	RYERSON UNIVERSITY
+ */
+
+public class Utils
+{
+	private static Logger logger = Logger.getLogger(ExportReport.class);
+	
+	public final static String	utf8					= "UTF-8";
```

Comment:
- Constants should be all upper case;
 - Consider reordering the modifiers to have "static" before "final", that's the expected order.